app/HutchManager/Controllers/ResultsController.cs

```diff
@@ -12,18 +14,38 @@ public class ResultsController: ControllerBase
 {
   private readonly ILogger<ResultsController> _logger;
   private readonly RQuestTaskApiClient _apiClient;
+  private readonly IFeatureManager _featureManager;
 
-  public ResultsController(ILogger<ResultsController> logger, RQuestTaskApiClient apiClient)
+  public ResultsController(ILogger<ResultsController> logger, RQuestTaskApiClient apiClient, IFeatureManager featureManager)
   {
     _logger = logger;
     _apiClient = apiClient;
+    _featureManager = featureManager;
   }
 
   [HttpPost]
   public async Task<IActionResult> Post([FromBody] QueryResult result)
```

Comment:
Can we document (`///`) that this is currently the RQuest route? helps explain why it's a 400 when `UseRoCrates` is true.